rules/windows/process_creation/proc_creation_win_susp_obfuscated_ip_via_cli.yml

```
@@ -17,10 +18,31 @@ detection:
         Image|endswith:
             - '\ping.exe'
             - '\arp.exe'
-    selection_ip:
-        - CommandLine|contains: ' 0x' # For hex format
-        - CommandLine|re: ' [0-9]{7,13}' # For octal format
-    condition: all of selection*
+    selection_ip_1:
+        CommandLine|contains:
+            - '//0x'
+            - '.0x'
+            - '.00x'
+    selection_ip_2:
+        CommandLine|contains|all:
+            - 'http://%'
+            - '%2e'
+    selection_ip_3:
+        # http://81.4.31754
+        - CommandLine|re: 'https?://[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,5}'
```

```suggestion
        - CommandLine|re: 'https?://[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,5}'
```
This can still match e.g. https://1.1.1